pmm-admin_test.go

```diff
@@ -266,20 +266,6 @@ func testConfigVerbose(t *testing.T, data pmmAdminData) {
 	assert.Equal(t, "< X-Remote-Ip: 127.0.0.1\n", cmdTest.ReadLine())
 	assert.Equal(t, "< \n", cmdTest.ReadLine())
 	assert.Equal(t, "< \"127.0.0.1:8300\"\n", cmdTest.ReadLine())
-	assert.Regexp(t, ".+ request:\n", cmdTest.ReadLine())
-	assert.Regexp(t, "> GET /v1/catalog/node/.+ HTTP/1.1\n", cmdTest.ReadLine())
-	assert.Regexp(t, "> Host: .+\n", cmdTest.ReadLine())
-	assert.Equal(t, "> User-Agent: Go-http-client/1.1\n", cmdTest.ReadLine())
-	assert.Equal(t, "> Accept-Encoding: gzip\n", cmdTest.ReadLine())
-	assert.Equal(t, "> \n", cmdTest.ReadLine())
-	assert.Equal(t, "> \n", cmdTest.ReadLine())
-	assert.Regexp(t, ".+ response:\n", cmdTest.ReadLine())
-	assert.Equal(t, "< HTTP/1.1 200 OK\n", cmdTest.ReadLine())
-	assert.Equal(t, "< Content-Length: 92\n", cmdTest.ReadLine())
-	assert.Equal(t, "< Content-Type: text/plain; charset=utf-8\n", cmdTest.ReadLine())
-	assert.Regexp(t, "< Date: .+\n", cmdTest.ReadLine())
-	assert.Equal(t, "< \n", cmdTest.ReadLine())
-	assert.Equal(t, "< {\"Node\":{\"ID\":\"\",\"Node\":\"\",\"Address\":\"\",\"TaggedAddresses\":null,\"Meta\":null},\"Services\":null}\n", cmdTest.ReadLine())
```

Comment:
This is interesting side effect.

When configuring client for the first time `pmm-admin` restarts all services... which means it restarts ZERO services because we just configured pmm-client so there is none. When we first configure pmm-client it shouldn't try to restart services as there are no services - buggy architecture. I'm not changing that now but because this PR removes call to consul when restarting all services we now have one less http call when configuring client for the first time.